.rspec

```diff
@@ -1 +1,4 @@
---color --require spec_helper
+--color
+--require spec_helper
+--format progress
+--format ParallelTests::RSpec::RuntimeLogger --out tmp/parallel_runtime_rspec.log
```

Comment:
These flags apply to non-parallel test runs as well, right? Would there be any effect in that case?

Reply:
Yeah, it will calculate the run times and update the log. Sounds harmless?

Reply:
I suppose it's fine.